### PR TITLE
docs: Added section on includes with Antora

### LINKED ISSUE #410: 'Skipping plantuml block' with plantuml block macro
### Wanted feature 

I would like to have my PlantUML-diagrams in seperate .puml-file, so I can reuse them in several documents. I first wrote a adoc with the plantuml inline and the works great.

After extracting an inline plantuml-diagram to a .puml-file and changing the adoc to use the [plantuml block macro](https://docs.asciidoctor.org/diagram-extension/latest/#diagram-macros) I get an error:

```
Skipping plantuml block macro. ENOENT: no such file or directory, open 'pumls/hello-world.puml'
```

### Reproduced

I have a git repo, which demonstrates this issue:

[https://github.com/mcbeelen/static-site-from-antora-with-search-and-diagrams/tree/broken/plantuml-block-macro](https://github.com/mcbeelen/static-site-from-antora-with-search-and-diagrams/tree/broken/plantuml-block-macro)

- `main-branch` is working
- `broken/plantuml-block-macro` is not working

Broken file is located at: `docs/modules/ROOT/pages/index.adoc`

### PR DESCRIPTION
**Pull request by mcbeelen**
Closes #410

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -189,6 +189,25 @@ include::alice-bob.puml[]
 
 ![PlantUML diagram](https://kroki.io/plantuml/png/eNpzKC5JLCopzc3hSszJTE5V0LVTSMpP4nJIzUsBCgIApPUKcg==)
 
+### References and includes with Antora
+
+If you use this Asciidoctor Kroki Extension in combination with Antora, all references and includes MUST use [Antora Resource IDs](https://docs.antora.org/antora/latest/page/resource-id/). The `.puml`-files are best placed in the [_partials_-directory](https://docs.antora.org/antora/latest/page/partials/) of the modules.
+
+#### Block macros
+
+```shell
```
Comment (ggrossetie) [inline README.md:198]:
```suggestion
```adoc
```

File: README.md [hunk 2]
```diff
@@ -189,6 +189,25 @@ include::alice-bob.puml[]
...
+vegalite::partial$chart.vlite[svg,role=chart,opts=interactive]
+```
+#### Includes
+
+```shell
```
Comment (ggrossetie) [inline README.md:203]:
```suggestion
```adoc
```

**Comment by ggrossetie** [comment]
Hey @mcbeelen thanks for this addition!
I left two nitpicks regarding the source code language which should be adoc (and not shell)

**Comment by mcbeelen** [comment]
@ggrossetie : Thanks for the review.
I updated the source language indicators.

**Pull request merged**